webpack/stories/docs/APIMiddleware.md

```diff
@@ -0,0 +1,46 @@
+# API Middleware
+
+Instead of each component handling API calls in the same way we have the API Middleware that will handle it instead.
+Current use as an action:
+
+API_OPERATIONS is defined in 'webpack/assets/javascripts/react_app/redux/API.js'
+```js
+import { API_OPERATIONS } from ...;
```

Comment:
It would be nice to give an example for plugins as well, e.g. `import {} from 'foremanReact/redux/API'`.

Reply:
Added